docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+app:
+  extends:
+    file: docker/common.yml
+    service: app
+  links:
+    - db
+  command: rackup -s puma -p 4567 --host 0.0.0.0
+  ports:
+    - "4567:4567"
```

Comment:
Ok, I've just added an `.env` variable that only docker-compose uses called `EXTERNAL_PORT` in ee72dcb6dd62c453fcb2e15a0a1ba36682f44fa5. You can use this to change the port that docker-compose exposes exercism at.